src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -17,26 +19,29 @@
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
+
+    public static final ArrayList<Meeting> EMPTYMEETINGLIST = new ArrayList<>();
```

Comment:
Perhaps this should be `EMPTY_MEETING_LIST` instead to follow the screaming snake case convention for constants